### DISCUSSION TITLE
Extra small breakpoint

### DISCUSSION THREAD

**Discussion by quick007** [tailwindlabs/tailwindcss#4157] [Ideas]
With multiple mobile devices under 362, I think that there should be a new breakpoint: xs. xs would be all devices over 362px. I'm mainly asking for this because of what you see below:
![image](https://user-images.githubusercontent.com/69743171/115782784-ecd55280-a370-11eb-9b19-ddaf6461496e.png)
![image](https://user-images.githubusercontent.com/69743171/115782859-04acd680-a371-11eb-896e-1faef3fa5946.png)

To be clear, those are all from the Galaxy fold, but making a breakpoint for just the galaxy fold seems a bit dumb, so I added 80px over the galaxy fold to include a number of other phones. I also wanted to make it useful to more people who don't care about the 3 people that use the galaxy fold and have it make more sense as a global breakpoint versus me just adding it to my config.

**Comment by quick007** [comment]
Those issues could probably be fixed with `break-all`, but I don't want to ruin the experience for more standard sized phones

**Comment by NikitaIT** [comment]
In truth, the division into screen sizes sm, md, lg itself is conceptually incorrect. This division is conceptual rather than real.

Most of the companies where I worked used the rule of setting media for each element separately.  This is already part of the specification [CSS Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries).

In your specific case, you just need the child blocks to take the width of the parent, this is your layout problem, not tw. You can use non-breaking space and `word-wrap: break-word`.
